serve.js

```diff
@@ -49,7 +50,7 @@ app.listen(program.port, () =>
     console.log(`Aspine listening on port ${program.port}!`)
 );
 
-if (program.secure) {
+if (!program.insecure) {
```

Comment:
There is no `program.insecure` because of how I implemented the `secure`/`insecure` (secure by default if and only if production). Instead, this can be changed to `if (program.secure || program.dev)` for the expected behavior with a plain `npm start` for development.